source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +21,12 @@ ReferenceCounter: class {
 		this _lock = Mutex new(mutexType)
 //		"  RC, #{This count}, #{this _target class name}" println()
 	}
+	free: override func {
+		This count -= 1
+//		"--RC, #{This count}, #{this _target class name}" println()
```

Comment:
Will you remove this comment and the one above while you're at it? They shouldn't be here